example/hydra/flake.nix

```diff
@@ -0,0 +1,124 @@
+{
+  description = "Cardano devnet with a Hydra head";
+
+  inputs = {
+    cardano-devnet.url = "path:../..";
+    flake-parts.follows = "cardano-devnet/flake-parts";
+
+    process-compose.url = "github:Platonic-Systems/process-compose-flake";
+
+    cardano-node.url = "github:IntersectMBO/cardano-node/10.1.4";
```

Comment:
Should we bump cardano-node to 10.5.3? The current version may be incompatible with Hydra v1.2.0